CHANGELOG.md

```diff
@@ -31,7 +31,8 @@ Release built: _not published yet_
 
 > [!CAUTION]
 > **Breaking Changes:**
-> - Changed ordering of the `/state/key-value-store/keys` endpoint. Entries are no longer orderer by their last modification state version but rather by their first appearance on the network, descending.
+> - Changed ordering of the collection returned by the `/state/key-value-store/keys` endpoint. Entries are no longer orderer by their last modification state version but rather by their first appearance on the network, descending. 
+> - Property `total_count` of the `/state/key-value-store/keys` endpoint is longer provided.
```

Comment:
I think you meant probably `is no longer provided`, and why we're no longer providing it?

Reply:
- Obviously you're right.
- We've lost it cause we moved away from the stored aggregates that provided this infomation to truly cursor-based pagination